extensions/wikia/Insights/Insights.setup.php

```diff
@@ -62,6 +62,22 @@
 $wgAutoloadClasses['InsightsModuleController'] = $IP.'/skins/oasis/modules/InsightsModuleController.class.php';
 
 /**
+ * Api Querys
+ */
+$wgAutoloadClasses[ 'ApiQueryUnconvertedInfoboxes' ] = $dir . 'ApiQueryUnconvertedInfoboxes.class.php';
+
+/**
+ * Query Pages
+ */
+$wgAutoloadClasses[ 'UnconvertedInfoboxesPage' ] = $dir . 'specials/UnconvertedInfoboxesPage.class.php';
+
+/**
+ * MW API
+ */
+$wgAPIListModules[ 'unconvertedinfoboxes' ] = 'ApiQueryUnconvertedInfoboxes';
```

Comment:
Should be moved to extensions/wikia/Insights/InsightsUnconvertedInfoboxes.setup.php